templates/workspace_services/guacamole/terraform/outputs.tf

```diff
@@ -1,7 +1,3 @@
-output "connection_uri" {
```

Comment:
Why is this no longer needed?

Reply:
Good question. Do you think there should be a connect button on the Guacamole workspace service, or just the VMs?

Reply:
I have found some use for that. As a workspace owner, it is convenient to access Guacamole's UI. However, I also recognize how it can create confusion for researchers.

Either way is fine with me.

Reply:
I'm going to take it out as causes confusion from a UX perspective.